tools/autopep8_extended.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import ast
+import re
+import sys
+
+import autopep8
+import inflection
+
+
+class Pep8Extended(object):
+    def __init__(self, pep8_options, source):
+        self.pep8_options = pep8_options
+        self.source = source
+
+    def get_checks(self):
+        msgs = {
+            'CW0001': 'Class name with snake_case style found, '
+                      'should use CamelCase. Change of "{old_class_name}" '
+                      'to "{new_class_name}" next (lines,columns): '
+                      '{lines_columns}',
+        }
+        return msgs
+
+    def check_cw0001(self):
+        code = 'CW0001'
+        msg = self.get_checks()[code]
+        class_renamed = {}
+        check_result = []
+
+        # Fix 'SyntaxError: encoding declaration in Unicode string'
```

Comment:
IMO you should split your method into smaller methods...
For exemple, this part of code removing the encoding comment found in first two lines should be externalized in an other method:  
```python
def strip_encoding():```
    """Remove encoding comment if found in first two lines
        If the first or second line has the `# xx coding xx` comment,
        it will be removed.
    """
```